### PR TITLE
feat(nfts): add support for nestable NFTs

### PR DESCRIPTION
**Pull request by tad3j**

### CODE REVIEW

File: packages/sdk/src/lib/common.ts [hunk 1]
```diff
@@ -2,6 +2,48 @@
...
 import axios from 'axios';
+import { ApillonApiError, ApillonNetworkError } from '../modules/apillon';
+
+/**
+ * Convert value to boolean if defined, else return undefined.
+ * @param value value converted
+ */
+export function toBoolean(value?: string) {
+  if (!value === undefined) {
+    return undefined;
```
Comment (tinemlakar) [inline packages/sdk/src/lib/common.ts:13]: This will never happen as `!undefined === true` !

File: packages/sdk/src/lib/common.ts [hunk 2]
```diff
@@ -2,6 +2,48 @@
...
+  }
+  return value === 'true' || value === '1';
```
Comment (tinemlakar) [inline packages/sdk/src/lib/common.ts:15]: `return value === 'true' || value == 1 || !!value;`

**Review by tinemlakar — CHANGES_REQUESTED** [review]

File: packages/cli/src/lib/files.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import fs from 'fs';
+
+export function readAndParseJson(filePath: string) {
+  let fileContent;
+  try {
+    fileContent = fs.readFileSync(filePath);
+  } catch (e) {
+    if (e.code === 'ENOENT') {
+      console.error(`Error: File not found (${filePath}).`);
+      return;
```
[the remark this answers is not in the document]
Comment (tad3j) [reply packages/cli/src/lib/files.ts:11]: I avoided it so we don't show stack trace to CLI user but just the error. Should we do that?

File: packages/cli/src/lib/types.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+export interface Params {
+  [Key: string]: string;
+}
+
+export interface Globals {
```
[the remark this answers is not in the document]
Comment (tad3j) [reply packages/cli/src/lib/types.ts:5]: `commander` calls them `optsWithGlobals` so I used that name. Will change.

**Review by tinemlakar — APPROVED** [review]

**Pull request merged**